blazor/pager/getting-started.md

```diff
@@ -248,9 +248,67 @@ For Blazor WebAssembly App, refer script in the `<head>` of the **~/index.html**
+        base.OnInitialized();
+        ListData.Add(new DataModel { Name = "Nancy, Berlin, France" });
+        ListData.Add(new DataModel { Name = "Andrew, Madrid, Germany" });
+        ListData.Add(new DataModel { Name = "Janet, London, Brazil" });
+        ListData.Add(new DataModel { Name = "Margaret, Marseille, Belgium" });
+        ListData.Add(new DataModel { Name = "Steven, Cholchester, Switzerland" });
+        ListData.Add(new DataModel { Name = "Laura , Tsawassen, Venezuela" });
+        ListData.Add(new DataModel { Name = "Robert, Tacoma, Austria" });
+        ListData.Add(new DataModel { Name = "Michael, Redmond, Mexico" });
+        ListData.Add(new DataModel { Name = "Albert, Kirkland, USA" });
+        ListData.Add(new DataModel { Name = "Nolan, London, Sweden" });
+        ListData.Add(new DataModel { Name = "Jennifer, Berlin, Finland" });
+        ListData.Add(new DataModel { Name = "Carter, Madrid, Italy" });
+        ListData.Add(new DataModel { Name = "Allison, Marseille, Spain" });
+        ListData.Add(new DataModel { Name = "John, Tsawassen, UK" });
+        ListData.Add(new DataModel { Name = "Susan, Redmond, Ireland" });
+        ListData.Add(new DataModel { Name = "Lydia, Cholchester, Portugal" });
+        ListData.Add(new DataModel { Name = "Kelsey, London, Canada" });
+        ListData.Add(new DataModel { Name = "Jessica, Kirkland, Denmark" });
+        ListData.Add(new DataModel { Name = "Robert, Berlin, Austria" });
+        ListData.Add(new DataModel { Name = "Shelley, Tacoma, Poland" });
+        ListData.Add(new DataModel { Name = "Vanjack, Tsawassen, Norway" });
+        ListData.Add(new DataModel { Name = "shelley, Cholchester, Argentina" });
+        ListData.Add(new DataModel { Name = "Lydia, Kirkland, Finland" });
+        ListData.Add(new DataModel { Name = "Jessica, Madrid, Sweden" });
+        ListData.Add(new DataModel { Name = "Nolan, London, UK" });
+        ListData.Add(new DataModel { Name = "Jennifer, Redmond, Italy" });
+    }
+    public class DataModel
+    {
+        public string Name { get; set; }
+        public string Id { get; set; }
+    }
+}
+```
+
+In the previous sample, list view items are rendered. The Pager component provides an option to split this list view data set into sectioned pages and view them page by page. So, now the Pager component is added in the **~/Pages/Index.razor** file under the **~/Pages** folder as the following.
+
+```cshtml
+@using Syncfusion.Blazor.Navigations
+
+<SfPager PageSize=5 NumericItemsCount=4 TotalItemsCount=25>
+</SfPager>
+```
+
+
+Based on the [PageSize]((https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfPager.html#Syncfusion_Blazor_Navigations_SfPager_PageSize)) property of the pager, list view items are bound to the current page. In the following code snippet, `PageSize` is defined as "5", so that the first five items from the data source of the list view will be displayed on the current page using the Skip and Take values.
```

Comment:
Based on the [PageSize]((https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfPager.html#Syncfusion_Blazor_Navigations_SfPager_PageSize)) property of the pager, the list view items are bound to the current page. In the following code sample, the `PageSize` is defined as "5" so that the first five items from the data source of the list view will be displayed on the current page using the Skip and Take values.